src/mavlink.cpp

```diff
@@ -416,37 +427,37 @@ void Mavlink::send_attitude(void)
 
 void Mavlink::send_imu(void)
 {
-//  if(RF_.sensors_.should_send_imu_data())
-//  {
-    mavlink_message_t msg;
-    vector_t accel = RF_.sensors_.data().accel;
-    vector_t gyro = RF_.sensors_.data().gyro;
-    mavlink_msg_small_imu_pack(sysid_, compid_, &msg,
-                               RF_.sensors_.data().imu_time,
-                               accel.x,
-                               accel.y,
-                               accel.z,
-                               gyro.x,
-                               gyro.y,
-                               gyro.z,
-                               RF_.sensors_.data().imu_temperature);
-    send_message(msg);
-//  }
-//  else
-//  {
-    // Otherwise, wait and signal that we still need to send IMU
-//    mavlink_streams[STREAM_ID_IMU].next_time_us -= mavlink_streams[STREAM_ID_IMU].period_us;
-//  }
+  //  if(RF_.sensors_.should_send_imu_data())
+  //  {
+  mavlink_message_t msg;
+  vector_t accel = RF_.sensors_.data().accel;
+  vector_t gyro = RF_.sensors_.data().gyro;
+  mavlink_msg_small_imu_pack(sysid_, compid_, &msg,
+                             RF_.sensors_.data().imu_time,
+                             accel.x,
+                             accel.y,
+                             accel.z,
+                             gyro.x,
+                             gyro.y,
+                             gyro.z,
+                             RF_.sensors_.data().imu_temperature);
+  send_message(msg);
+  //  }
+  //  else
+  //  {
+  // Otherwise, wait and signal that we still need to send IMU
+  //    mavlink_streams[STREAM_ID_IMU].next_time_us -= mavlink_streams[STREAM_ID_IMU].period_us;
+  //  }
```

Comment:
Yeah, that is on purpose (for now).   That little routine is to make sure that we don't send the same IMU measurement more than once.